### ISSUE TITLE
Add line number reference to all error reports

### ISSUE THREAD

**Issue by qtpmgrossman** [SeleniumHQ/selenium-ide#852]
## 🚀 Feature Proposal

Optimize the debugging process by including the line of code where a defect is detected.

## Motivation

I recently recorded an extensive script entering data into multiple fields.
At some point a Type command without any input data was generated.

When I attempted to export the code in all three programming languages, this error was blocked the process:  "Incomplete command 'type'. Missing expected value argument."

20 Type lines were reviewed until the one missing the input was detected.  
It would be quicker to detect the problematic line of code if the line number was included in all error reports.   

## Example

While the issue was generated in a recording, the steps to reproduce are difficult to replicate. 
This is a manual replication:

1) Have one developer record an extensive script with data input fields.
2) Clear the data from one random Type line. Save.
3) Have a second developer attempt to export and debug the error. 
4) Note the amount of time to debug issue.

5) Repeat the steps above, but tell the second developer which line was modified.
6) Note difference in debugging time.

Please provide an example for how this feature would be used.

Just more efficient debugging for all users.

**Comment by tourdedave** [comment]
Agreed, this would be a nice upgrade. Unfortunately, at the moment, the position information isn't easily available on export. We would need to pass the exported tests through the same mechanism used for playback (which constructs a syntax tree). From that, we could derive the step # and have more helpful error messages.

**Comment by hchiam** [comment]
It'd be nice if the CLI output and results file output would just say the step that it errored at, instead of making the tester have to do a "binary search" through a long test.
